src/screens/OrgList/OrgList.tsx

```diff
@@ -105,6 +108,20 @@ function orgList(): JSX.Element {
     notifyOnNetworkStatusChange: true,
   });
 
+  const [displayedOrgs, setDisplayedOrgs] = useState(
+    orgsData?.organizationsConnection || []
+  );
+
+  useEffect(() => {
+    if (orgsData && orgsData.organizationsConnection) {
+      const newDisplayedOrgs = sortOrgs(
```

Comment:
Sorting of organizations should happen on server side and not client side

Reply:
@rishav-jha-mech , could you please provide any reference where useQuery params are modified to get the sorted data according to createdAt_ASC or createdAt_DESC .